knowageutils/src/main/java/it/eng/knowage/commons/security/PathTraversalChecker.java

```diff
@@ -37,6 +37,25 @@ private PathTraversalChecker() {
 		throw new IllegalStateException("This class provides utility methods. It cannot be instantiated");
 	}
 
+	public static void get(String firstDirectory, String... otherFolders) throws PathTraversalAttackException {
+		File previousFolderFile = new File(firstDirectory);
+
+		for (String currentFolder : otherFolders) {
+			try {
+				isValidFileName(currentFolder);
+
+				File currentFolderFile = new File(currentFolder);
+
+				preventPathTraversalAttack(currentFolderFile, previousFolderFile);
+
+				previousFolderFile = currentFolderFile;
+			} catch (Exception e) {
+				logger.error(e);
+				throw new PathTraversalAttackException(e.getMessage());
+			}
```

Comment:
If the code is not throwing any exception (apart from runtime PathTraversalAttackException), we can remove the try catch block